operators/pkg/controller/elasticsearch/elasticsearch_controller.go

```diff
@@ -233,6 +233,10 @@ func (r *ReconcileElasticsearch) internalReconcile(
 		return results
 	}
 
+	if err := r.handleDynamicWatches(es); err != nil {
```

Comment:
Related to my comment above: I would move that into the http cert reconciliation function.